network/p2p/unicast/ratelimit/rate_limiters.go

```diff
@@ -80,13 +79,13 @@ func (r *RateLimiters) MessageAllowed(peerID peer.ID) bool {
 
 // BandwidthAllowed will return result from BandWidthRateLimiter.Allow. It will invoke the OnRateLimitedPeerFunc
 // callback each time a peer is not allowed.
-func (r *RateLimiters) BandwidthAllowed(peerID peer.ID, role string, msg *message.Message) bool {
```

Comment:
sticking to the Least Privilege Principle, let's only feed the information that this component needs.